lib/ASTDecl.cpp

```diff
@@ -60,27 +63,43 @@ namespace pillar
         assert(body.getBlocks().empty());
         return func_decl;
       }
+      AST::lift_queue->emplace_back([&body, func_decl, functionName, this](void)
```

Comment:
Remove `functionName`, not needed in here. You can shorten this as: `[&body, =, this]`

Reply:
Extract this lambda into a local var; it'll keep the indentation down. E.g.

```c++
auto lift_body = [&body, =, this] (void) {

};
lift_queue.emplace_back(std::move(lift_body));
```